helm/libero-editor/templates/deployment-article-store.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "libero-editor.fullname" . }}-article-store
+  labels:
+    app.kubernetes.io/name: {{ include "libero-editor.name" . }}
+    helm.sh/chart: {{ include "libero-editor.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: article-store
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
You no longer have a general replica count. I think this needs to be .Values.articleStore.replicaCount

Reply:
Good spot, shame 'helm lint' can do the same :-)